src/app/hooks/modal.ts

```diff
@@ -3,8 +3,8 @@ import { parse as parseQueryString, stringifyUrl } from 'query-string';
 import { useAsyncTermsReview } from '@app/services/termsReview';
 import { getTermsAppURL } from '@app/utils/termsApp';
 
-export const useModalControl = () => {
-    const loadTermsReview = useAsyncTermsReview();
+export const useModalControl = (eventCode: string) => {
```

Comment:
do the same for the siteCode - though you can pass a single type

Reply:
done